lib/services/oauth/oauth_server.go

```diff
@@ -2,9 +2,13 @@ package oauth
 
 import (
 	"context"
+	"fmt"
 	"os"
 
+	"github.com/markgravity/golang-ic/database"
+	"github.com/markgravity/golang-ic/helpers"
 	"github.com/markgravity/golang-ic/helpers/log"
+	models2 "github.com/markgravity/golang-ic/lib/models"
```

Comment:
```suggestion
	"github.com/markgravity/golang-ic/lib/models"
```

Let's never use auto-incremented names for module imports 🙈 

As the whole application will use `lib/models`, it might be nice to keep it `models`.
In this case, `"github.com/go-oauth2/oauth2/v4/models"` is the exception, so we should probably alias that one to `oauthmodels` ? :)

Reply:
Also it could be nice to have translations for the error messages 🙇

Reply:
@malparty no more message for translating after changes :D